packages/docs/src/pages/en/features/theme.md

```diff
@@ -9,6 +9,12 @@ related:
   - /getting-started/wireframes/
 ---
 
+<script setup>
```

Comment:
As per your previous PR's, please remove this and the tabs/window code.
We do need to support TS and JS simultaneously, but it still needs to be implemented at a global scale (which is outside of the scope of just this file) For the time being, it should suffice to put the JS and TS code blocks next to each other

Reply:
This should be resolved sorry for my very bad memory and thank you for reminding me once again.